initializers/rack_groups.yml

```diff
@@ -0,0 +1,3 @@
+#- name: cage 101
+#  slug: cage-101
+#  site: site01
```

Comment:
there is no `site01` in the default scripts. Can you please change it to an existing site, e.g. `SING 1`?

Reply:
Done, I also added a rack from site `SING 1` to the example `cage 101` rack group.